test/test_arraycontext.py

```diff
@@ -114,147 +113,10 @@ def _acf():
 # }}}
 
 
-# {{{ stand-in DOFArray implementation
-
-@with_container_arithmetic(
```

Comment:
This doesn't need to move anymore, right? Since it doesn't have any string annotations.

I'm in favor of the little `testlib` regardless, so just curious :grin:

Reply:
If it stood on its own, yes, it would not need to move. But there are things in `testlib` that depend on it, and it's not importable here. It also just seems happier there.

Reply:
Ah, right, missed that.